src/cosalib/meta.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, workdir=None, build='latest',
         # Load the schema
         self._validator = None
         self._schema_path = schema
-        if schema:
+        if schema and schema.lower() not in ("false", "none"):
```

Comment:
Do we need this? Can we just do `if schema:`, which catches both the `None` case and if it's been explicitly disabled by setting to `""`?

Reply:
The rationale is that `--schema=none` is better than `--schema=`

Reply:
Yeah, that's OK by me.

Reply:
OK, I got confused here and thought it was fed via the env var. But right, I see now both the env var or the switch can affect it.